crnk-meta/src/main/java/io/crnk/meta/internal/typed/MetaDataObjectProviderBase.java

```diff
@@ -31,7 +32,9 @@ protected void createAttributes(T meta) {
 		for (String name : beanInformation.getAttributeNames()) {
 			BeanAttributeInformation attrInformation = beanInformation.getAttribute(name);
 			if (attrInformation.getGetter() != null && !isIgnored(attrInformation)) {
-				if (attrInformation.getGetter().getDeclaringClass() != implClass) {
+				boolean getterDeclaredInImplClass = attrInformation.getGetter().getDeclaringClass() == implClass;
+				boolean generic = attrInformation.getField().getGenericType() instanceof TypeVariable;
+				if (!getterDeclaredInImplClass && !generic) {
```

Comment:
I'm a bit undecided about this one. The direction is good. But not sure, I think we can rather not make a new "declarted" MetaAttribute in the subtype. Somehow it should rather be reflected in the "attributes" rather than "declaredAttributes" list.